src/components/badge/README.md

```diff
@@ -1 +1,44 @@
-# Badge
\ No newline at end of file
+# Badge
+
+Use badges to call out or label important information in your UI.
+
+>
+    Concise labels
+
+    Because badges are intended to call attention to information, it is important to keep badge text concise. It should not span multiple lines.
+
+## When to use
+
+- To label new content on a page
+- To help users discover important information changes in your UI
+
+## When to consider something else
+
+- When there is a critical error that requires the users attention to fix, use a page-level alert instead
+- Avoid using lots of badges on one page when possible as it will diminish the effect of calling out important parts of the UI
+
+## Adding the markup
+
+### Default badges
+
+```
+<span class="rvt-badge">Base</span>
+<span class="rvt-badge rvt-badge--info">Info</span>
+<span class="rvt-badge rvt-badge--success">Success</span>
+<span class="rvt-badge rvt-badge--warning">Warning</span>
+<span class="rvt-badge rvt-badge--danger">Danger</span>
+```
+
+### Secondary badges
+
+```
+<span class="rvt-badge rvt-badge--secondary">Base</span>
+<span class="rvt-badge rvt-badge--info-secondary">Info</span>
+<span class="rvt-badge rvt-badge--success-secondary">Success</span>
+<span class="rvt-badge rvt-badge--warning-secondary">Warning</span>
+<span class="rvt-badge rvt-badge--danger-secondary">Danger</span>
+```
```

Comment:
It was having trouble handling a suggestion this complex:

## Adding the markup

Rivet badges consist of a `span` element with the class `rvt-badge`.

```
<span class="rvt-badge">Base</span>
```

### Additional options

#### Secondary

By default, Rivet badges are black text on a grey background. The addition of the `rvt-badge--secondary` modifier class produces a badge with a grey border, white background, and black text.

```
<span class="rvt-badge rvt-badge--secondary">Secondary</span>
```

#### Information

Rivet provides additional options that allow you to convey information via badges.

```
<span class="rvt-badge rvt-badge--info">Info</span>
<span class="rvt-badge rvt-badge--success">Success</span>
<span class="rvt-badge rvt-badge--warning">Warning</span>
<span class="rvt-badge rvt-badge--danger">Danger</span>
```

These are stackable with secondary badges as well.

```
<span class="rvt-badge rvt-badge--info-secondary">Info</span>
<span class="rvt-badge rvt-badge--success-secondary">Success</span>
<span class="rvt-badge rvt-badge--warning-secondary">Warning</span>
<span class="rvt-badge rvt-badge--danger-secondary">Danger</span>
```